utilities/test_suite/HIP/runAudioTests.py

```diff
@@ -224,7 +223,7 @@ def rpp_test_suite_parser_and_validator():
 subprocess.call(["make", "-j16"], cwd=".")    # nosec
 
 # List of cases supported
-supportedCaseList = ['0', '1', '3']
+supportedCaseList = ['0', '1', '3', '7']
```

Comment:
What about these updates for the HOST backend python script?
https://github.com/ROCm/rpp/blob/develop/utilities/test_suite/HOST/runAudioTests.py

Reply:
HOST is already in ToT, this PR is only for HIP support.
HOST L179 -> https://github.com/ROCm/rpp/blob/develop/utilities/test_suite/HOST/runAudioTests.py#L179